Mail/Views/Thread/Message/Attachment/AttachmentsView.swift

```diff
@@ -113,34 +143,52 @@ struct AttachmentsView: View {
             DocumentPicker(pickerType: .exportContent(attachmentsURL.urls))
                 .ignoresSafeArea()
         }
+        .onDisappear {
+            for (_, task) in trackDownloadTask {
+                task.cancel()
+            }
+        }
```

Comment:
So this code doesn't work, but I think that's normal as the view is called in the ***MessageSubHeaderView*** which is called in the ***MessageView***.